microk8s-resources/actions/enable.cilium.sh

```diff
@@ -0,0 +1,94 @@
+#!/usr/bin/env bash
+
+set -e
+
+source $SNAP/actions/common/utils.sh
+
+ARCH=$(arch)
+if ! [ "${ARCH}" = "amd64" ]; then
+  echo "Cilium is not available for ${ARCH}" >&2
+  exit 1
+fi
+
+"$SNAP/microk8s-enable.wrapper" helm
+
+echo "Restarting kube-apiserver"
+refresh_opt_in_config "allow-privileged" "true" kube-apiserver
+sudo systemctl restart snap.${SNAP_NAME}.daemon-apiserver
+
+echo "Enabling Cilium"
+
+if [ ! -f "${SNAP_DATA}/bin/cilium" ]
+then
+  CILIUM_VERSION="v1.6"
```

Comment:
I see you have hard coded the version here. It is fine with me. In case you want to allow more flexibility users could provide the version while they enable the add-on. In linkerd for example you can do  `microk8s.enable "linkerd:proxy-auto-inject;tls=optional;skip-outbound-ports=1234,3456"` see https://github.com/ubuntu/microk8s/blob/feature/insecure-port/microk8s-resources/actions/enable.linkerd.sh#L7

Reply:
Oh, that's cool. The only trick is that this name has to be the same as a branch name on the Cilium repository. For my local testing, I've just been installing using these scripts, then manually editing the image tag to change it. This could be useful though.